table.cc

```diff
@@ -2542,6 +2543,9 @@ future<row_locker::lock_holder> table::do_push_view_replica_updates(const schema
     auto& base = schema();
     m.upgrade(base);
     gc_clock::time_point time_point = gc_clock::now();
+    utils::get_local_injector().inject("table_push_view_replica_updates_stale_time_point", [&time_point] {
+        time_point -= 10s;
```

Comment:
I don't understand how this injection makes a difference.... Unless I'm missing something, it's identical to not messing with time_point, and instead sleeping for 10 seconds (so after the sleep, the timepoint is 10 seconds in the past). Since sleeping 10 seconds should be fine here, as anywhere, what can this injection discover?
Did you reproduce any bug using this injection?

Reply:
I didn't reproduce anything (since this series works on top of a commit which unifies the time points here), but the idea is to create and check a scenario in which a value expires within these 10 seconds due to its TTL - and we could verify that we didn't end up with inconsistencies, even if on one node the check was done before a value expired, and on the other - the opposite happened.